Examples/ElizaSharedSources/GeneratedSources/eliza.connect.swift

```diff
@@ -41,16 +41,16 @@ internal final class Connectrpc_Eliza_V1_ElizaServiceClient: Connectrpc_Eliza_V1
 
     @available(iOS 13, *)
     internal func `say`(request: Connectrpc_Eliza_V1_SayRequest, headers: Connect.Headers = [:]) async -> ResponseMessage<Connectrpc_Eliza_V1_SayResponse> {
-        return await self.client.unary(path: "/connectrpc.eliza.v1.ElizaService/Say", request: request, headers: headers)
```

Comment:
These changes need to be reverted, that's why CI is failing

Reply:
Thanks! Done.